presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
+            throws Exception
+    {
+        try {
+            TypedSet typedSet = new TypedSet(BIGINT, 10, FUNCTION_NAME);
+
+            for (int i = 0; i <= TypedSet.FOUR_MEGABYTES + 1; i++) {
+                Block block = createLongsBlock(Collections.nCopies((int) 1, (long) i));
+
+                typedSet.add(block, 0);
+            }
+
+            fail("expected exception");
```

Comment:
yea, i have this kind of silly but mechanical rule for blank lines:
1. before any if/else/while/try... construct
2. after any }  that contains any code on it except when that is part of a chain (if-else, try-catch. ...)
3. before / after any block of var decs

and that's pretty much it. it keeps me from having to think about whether this case or that case requires a blank line. i just follow those rules and think about other stuff :)
(I'll admit sometimes it produces cases that may look funny, but it's a tradeoff for objective consistency)

lmk which of these aren't compatible and how you'd like them changed